packages/flutter_tools/test/integration.shard/test_data/deferred_components_project.dart

```diff
@@ -186,7 +186,7 @@ class BasicDeferredComponentsConfig extends DeferredComponentsConfig {
       }
 
       defaultConfig {
-          // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+          // Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
```

Comment:
@GaryQian

Reply:
this is output text that is for the developer

Reply:
Same thing here, do we really need to use a todo if this is not meant for us?

Reply:
yes